interpretation/ww_head.py

```diff
@@ -40,7 +40,7 @@ def process(message, isGameMaster=False, isAdmin=False, isPeasant=False):
     # =============================================================
     if isPeasant == True:
         if is_command(message,['warn'],False,unip):
-            # Warn the user that they've been active for a while.
+            # Warn the user that they've been inactive for a while.
```

Comment:
Belongs to a different commit, but ok

Reply:
I just saw that and thought it was weird, so changed it